docs/pages/programming/jdbc_api.md

```diff
@@ -65,6 +83,20 @@ An uber JAR is a JAR file which contains all its dependencies. To use the JAR wi
 export CLASSPATH=<path_to_the_uber_jar>:$CLASSPATH
 ```
 
+### With Gradle
+
+To introduce cdb2jdbc in your applications as a Maven dependency, add the following to `build.gradle`, with the version available from your Maven repository.
```

Comment:
> To introduce cdb2jdbc in your applications as a Maven dependency

Should it be "To introduce cdb2jdbc in your applications as a *Gradle* dependency"?

Reply:
Ah it's still a maven dependency in gradle. Gradle can depend on other repositories such as Ivy for example. I can clarify this.